packages/asset-buyer/src/order_providers/standard_relayer_api_order_provider.ts

```diff
@@ -111,4 +111,32 @@ export class StandardRelayerAPIOrderProvider implements OrderProvider {
             }
         });
     }
+    /**
+     * Given a maker asset data string, return all availabled paired taker asset data strings.
+     * @param   makerAssetData   A string representing the maker asset data.
+     * @return  An array of asset data strings that can be used to purchased makerAssetData.
+     */
+    public async getAvailableTakerAssetDatasAsync(makerAssetData: string): Promise<string[]> {
+        // Return a maximum of 1000 asset datas
+        const maxPerPage = 1000;
+        const requestOpts = { networkId: this.networkId, perPage: maxPerPage };
+        const assetPairsRequest = { assetDataA: makerAssetData };
+        const fullRequest = {
+            ...requestOpts,
+            ...assetPairsRequest,
+        };
+        let response: AssetPairsResponse;
+        try {
+            response = await this._sraClient.getAssetPairsAsync(fullRequest);
+        } catch (err) {
+            throw new Error(AssetSwapQuoterError.StandardRelayerApiError);
```

Comment:
Are we sure we want to just swallow the actual error returned? Might make it harder to debug the actual issue. Perhaps we log it?